src/components/CreateVault.tsx

```diff
@@ -1,44 +1,49 @@
 import { useCallback } from 'react';
 import { useVaultStore, ViewMode, viewModeAtom } from 'store/vaults';
 import { useSetAtom } from 'jotai';
-import CollateralChoice from './CollateralChoice';
+import CollateralChoice from 'components/CollateralChoice';
+import ConfigureNewVault from 'components/ConfigureNewVault';
+import NewVaultOfferSummary from 'components/NewVaultOfferSummary';
 
-const CollateralChoices = () => {
+const CreateVault = () => {
   const setMode = useSetAtom(viewModeAtom);
   const { vaultManagerIds } = useVaultStore();
 
   const buttonProps = {
-    text: 'Back to Manage Vaults',
+    text: 'Cancel',
     onClick: useCallback(() => setMode(ViewMode.Manage), [setMode]),
   };
 
   return (
     <>
-      <div className="flex justify-between mt-6">
+      <div className="flex justify-between mt-6 flex-wrap gap-4">
         <div className="font-serif font-medium text-2xl">
           Creating New Vault
         </div>
         <button
-          className="text-[#f9fafe] text-xs uppercase flex flex-row justify-center items-center p-3 bg-interPurple rounded-md shadow-[0_10px_14px_-4px_rgba(183,135,245,0.3)]"
+          className="transition mr-1 text-[#A3A5B9] uppercase rounded-[6px] border-2 border-solid border-[#A3A5B9] py-2 px-7 leading-[14px] font-semibold text-xs bg-[#A3A5B9] bg-opacity-0 hover:bg-opacity-10 active:bg-opacity-20"
           onClick={buttonProps.onClick}
         >
           {buttonProps.text}
         </button>
       </div>
-      <div className="mt-16 grid grid-cols-11 gap-x-8">
-        <div className="col-span-7">
+      <div className="mt-16 grid grid-cols-11 lg:gap-x-[51px]">
```

Comment:
choosing gaps that don't match the grid rhythm is suspect to me. do we really need 51px instead of an existing named value?

maybe because the gap below is 18.5px. maybe out of scope to address.